FryProxy/src/Utils/DnsUtils.cs

```diff
@@ -12,7 +12,7 @@ namespace FryProxy.Utils
     /// </summary>
     public static class DnsUtils
     {
-        private static readonly Regex HostAndPortRegex = new Regex(@"(?<host>\w+):(?<port>\d+)");
+        private static readonly Regex HostAndPortRegex = new Regex(@"(?<host>.+):(?<port>\d+)");

```

Comment:
Please apply following rules:
> Valid characters for hostnames are ASCII(7) letters
       from a to z, the digits from 0 to 9, and the hyphen (-).  A hostname
       may not start with a hyphen.

Reply:
In regex it would be like: `@"(?<host>[\w-\.]+):(?<port>\d+)"`

Reply:
updated